db/1.10/modules/ROOT/pages/database-connector-transactions.adoc

```diff
@@ -74,6 +74,11 @@ In the XML editor, the `transactionalAction` configuration looks like this:
 </try>
 ----
 
+== Transactions and Streaming
+
+Results from some Database Connector operations (for example the *Select* operation) are automatically streamed to prevent loading all the results at once, which can lead to performance and memory issues. This behavior means that database rows are loaded lazily when needed.
```

Comment:
Is "lazily" a term users will know? I never encountered it. Maybe you can remove it here and elsewhere.